### PR TITLE
ADEN-1800 Adding Turtle

### PR DESCRIPTION
**Pull request by gabrys**

### CODE REVIEW

File: extensions/wikia/AdEngine/AdEngine2ContextService.class.php [hunk 1]
```diff
@@ -71,6 +71,7 @@ public function getContext( Title $title, $skinName ) {
 				'forceProviders' => $this->filterOutEmptyItems( [
 					'directGpt' => $wg->AdDriverForceDirectGptAd,
 					'liftium' => $wg->AdDriverForceLiftiumAd,
+					'turtle' => !!$wg->AdDriverForceTurtleAd,
```
Comment (nandy-andy) [inline extensions/wikia/AdEngine/AdEngine2ContextService.class.php]: Why are we forcing it to be `Boolean` and other (above) aren't casted? Shouldn't we cast all of them just in matter of consistency?

**Comment by nandy-andy** [comment]
Except my curiosity connected to small inconsistency the changes look good :+1:

File: extensions/wikia/AdEngine/AdEngine2ContextService.class.php (hunk 1 above)
Comment (gabrys) [inline extensions/wikia/AdEngine/AdEngine2ContextService.class.php]: Ha! Good catch. No reason for that. We can either cast them or not cast them. There's no point to do both. Sorry about it. The reason is I moved the line from the block where we cast vars to bools to the block where we don't.

**Comment by nandy-andy** [comment]
LGTM :+1:

**Pull request merged**